### PR TITLE
Added version file. #39224

### PR DESCRIPTION
**Pull request by asifhj**
This merge is for https://github.com/kubernetes/kubernetes/issues/39224, https://github.com/kubernetes/kubernetes/issues/10082, https://github.com/kubernetes/kubernetes/issues/43221, which includes version file.

### CODE REVIEW

File: cluster/common.sh [hunk 1]
```diff
@@ -20,12 +20,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(cd $(dirname "${BASH_SOURCE}")/.. && pwd)
```
Comment (ixdy) [inline cluster/common.sh]: why did you delete this?

File: version [hunk 2]
```diff
@@ -0,0 +1 @@
+v1.5.4
```
Comment (ixdy) [inline version]: The `version` file is auto-generated by the build process. It shouldn't be checked into the repo.

File: cluster/common.sh [hunk 3]
```diff
@@ -20,12 +20,17 @@ set -o errexit
...
-
 DEFAULT_KUBECONFIG="${HOME}/.kube/config"
 
-source "${KUBE_ROOT}/cluster/lib/util.sh"
-source "${KUBE_ROOT}/cluster/lib/logging.sh"
+if [ -f "${KUBE_ROOT}/cluster/lib/util.sh" ]; then
```
Comment (ixdy) [inline cluster/common.sh]: I don't think this is necessary. It seems more likely that `KUBE_ROOT` is being set incorrectly somewhere.

**Comment by ixdy** [comment]
@asifhj can you provide more details on what you're trying to fix here? I'm missing the context for what this PR is attempting to do.

File: version (hunk 2 above)
Comment (asifhj) [inline version]: This is required if you installing following manual installation guide.
https://kubernetes.io/docs/getting-started-guides/ubuntu/manual/

Agree with you build adds this file but when you do manual installation this file should exist.

File: cluster/common.sh (hunk 3 above)
Comment (asifhj) [inline cluster/common.sh]: KUBE_ROOT is getting set to wrong path, and same path is used in both Master and Minion node.

File: cluster/common.sh (hunk 1 above)
Comment (asifhj) [inline cluster/common.sh]: KUBE_ROOT is getting set to wrong path, and same path is used for both Master and Minion node.

**Comment by asifhj** [comment]
@ixdy This fix is for  #39224, #10082, #43221

**Comment by ixdy** [comment]
per the discussion on #39224, I don't think we should move forward with this PR.

**Comment by ixdy** [comment]
closing in favor of #44344, which is deleting the ubuntu provider entirely. (use kubeadm instead!)

cc @mikedanese
